.github/workflows/dependency-review.yml

```diff
@@ -49,3 +49,4 @@ jobs:
           # Example: advanced-security/reusable-workflows/.github/dependency-review.yml@main
           config-file: ${{ steps.config.outputs.config }}
           comment-summary-in-pr: "always"
+          retry-on-snapshot-warnings: "true" #This will add a 120s delay on any repos without a snapshot
```

Comment:
Todo:  parameterize  - otherwise this will add 120s delay by default while it spin waits for a snapshot every 10s until 120s

Reply:
So what happens if a snapshot never comes? Like for a Python or JS project